### ISSUE TITLE
Remove deprecated twisted.spread.refpath

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5482]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#5482|
|Type|task|
|Created|2012-02-13 09:37:24Z|
|Branch|https://github.com/twisted/twisted/tree/spread-refpath-5482|

[twisted/spread/refpath.py](twisted/spread/refpath.py) was deprecated in Twisted 9.0 (r26694/[#3723](https://github.com/twisted/twisted/issues/3723)) and can be removed.

<details><summary>Searchable metadata</summary>

```
trac-id__5482 5482
type__task task
reporter__thijs thijs
priority__lowest lowest
milestone__ 
branch__branches_spread_refpath_5482 branches/spread-refpath-5482
branch_author__thijs thijs
status__closed closed
resolution__fixed fixed
component__core core
keywords__easy easy
time__1329125844000000 1329125844000000
changetime__1329844650000000 1329844650000000
version__None None
owner__thijs thijs
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra commented|
|-|-|

(In [33563]) Branching to 'spread-refpath-5482'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra commented|
|-|-|

(In [33564]) remove refpath, refs [#5482](https://github.com/twisted/twisted/issues/5482)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra removed owner<br>@thijstriemstra set status to `new`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)<a name="note_5"></a>|@itamarst set owner to @thijstriemstra|
|-|-|

Looks good, merge.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_6"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [33599]) Merge spread-refpath-5482: Remove t.spread.refpath, deprecated since 9.0.

Author: thijs
Reviewer: itamar
Fixes: [#5482](https://github.com/twisted/twisted/issues/5482)
